Google Meet/disc/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "author": {
+      "name": "Drull",
+      "id": "244953147154366464"
+    },
+    "url": "meet.google.com",
+    "description": {
+      "en": " Google Meet is a communication platform, developed by Google, which includes instant messaging, video chat, SMS and VOIP.",
+      "pt_BR": "Google Meet é uma plataforma de comunicação, desenvolvida pela Google, que inclui mensagens instantâneas, chat de vídeo, SMS e VOIP."
+    },
+    "service": "Google Meet",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/OFbN8Uu.png",
+    "thumbnail": "https://i.imgur.com/6BRZ63d.png",
+    "color": "#308574",
+    "tags": [
+      "google",
+      "meet",
+      "video"
+    ],
+    "category": "socials",
+    "warning": true,
```

Comment:
what's the reason to use warning?

Reply:
I renamed the dist file and also removed this warning part of the code.

Reply:
I created another file called dist and deleted the old one called "disc"